proto/ots/bettor.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package ots;
+
+option go_package = "oddin.gg/ots";
+option java_package = "com.oddin.ots";
+
+import "google/protobuf/timestamp.proto";
+import "ots/commons.proto";
+
+message BettorRiskCoefficientsRequest {
+  // if recover_since is null, no updates will be recovered. If it is defined we will start to send to consumer
+  // all BettorRiskCoefficients since given timestamp value
+  optional google.protobuf.Timestamp recover_since = 1 ;
+}
+
+message BettorRiskCoefficientsResponse {
```

Comment:
name: Response. It is not a response. But I understand that you tried to follow the current naming. But there is TicketResponse and it is really response or TicketResultResponse which is not so much response :D 

I would rename it to: `BettorEvent`.

Reply:
I understand but when we apply all rename suggestions, gRPC endpoint will be:

```
rpc Bettors(BettorsRequest) returns (stream BettorEvent);
```

That's against the definition of the others. I think it is better to stay consistent so:

```
rpc Bettors(BettorsRequest) returns (stream BettorsResponse);
```

What do you think?

Reply:
After discussion with Pavel we decided to keep Response and follow current schema naming.